fix: add to naj faq (#1148)
near/docs: docs/4.tools/near-api-js/faq.md

---

> ### What frontend frameworks can I use the JavaScript API with?
> The JavaScript API is framework-agnostic. You can include it in any frontend frameworks, such as React, Vue, Angular and others.

Reviewer: ```suggestion
### What front-end frameworks can I use the JavaScript API with?

The JavaScript API is framework-agnostic. You can include it in any front-end framework, such as React, Vue, Angular, and others.
```

You can use [`create-near-app`](https://github.com/near/create-near-app) to quickly bootstrap projects with different templates:

    npx create-near-app

### Can I use the JavaScript API with mobile JavaScript frameworks such as React Native?

> The JavaScript API can be used in most JavaScript runtimes, and under the hood it’s an abstraction over Near’s [RPC API](/api/rpc/introduction). However, notice that the Wallet can’t be used everywhere. For example, in React Native apps you’ll be able to use the Wallet in web versions of the apps, but it won’t work in the native app deployments.

Reviewer: ```suggestion
The JavaScript API can be used in most JavaScript runtimes, and under the hood, it’s an abstraction over NEAR’s [RPC API](/api/rpc/introduction). However, notice that the Wallet can’t be used everywhere. For example, in React Native apps you’ll be able to use the Wallet in web versions of the apps, but it won’t work in the native app deployments.
```

You can use the Wallet in `WebView` components in iOS or Android, however be aware that it uses `LocalStorage` for `KeyStore`, and it’s your responsibility to persist the storage when you manage loading of `WebView` components.

---

## Transactions {#transactions}

### How to check the status of transaction
Please refer to examples about transactions in the [Cookbook](/tools/near-api-js/cookbook).

### How to send batch transactions
Please refer to examples about transactions in the [Cookbook](/tools/near-api-js/cookbook).

---

## Accounts {#accounts}

> ### What’s the difference between `Account` and `ConnectedWalletAccount`

Reviewer: ```suggestion
### What’s the difference between `Account` and `ConnectedWalletAccount`?
```

> Interaction with the wallet is only possible in a web-browser because Near’s Wallet is web-based.

Reviewer: ```suggestion
Interaction with the wallet is only possible in a web-browser environment because NEAR’s Wallet is web-based.
```
